_component_design/toast.md

```diff
@@ -98,22 +114,22 @@ Icons are always centered vertically.
 
 <section class="static-section" markdown="1">
 
-## Variations
-
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Without a CTA
+### Style
 
-Clicking on the “X” dismisses the toast notification. There are no other actions that can be taken on this type of toast notification.
+There are three types of toast notifications: Success/Confirmation, Informational, and Error.
```

Comment:
The three types of toast notifications are Success/Confirmation, Informational, and Error.

Reply:
Done.